youngseo/.env.sample

```
@@ -0,0 +1,9 @@
+PORT = YOUR PORT NUMBER SHOULD BE HERE!
+
+TYPEORM_CONNECTION = mysql
+TYPEORM_HOST = HOST NUMBER
+TYPEORM_USERNAME = root
+TYPEORM_PASSWORD = myPassword
+TYPEORM_DATABASE = myDB
+TYPEORM_PORT = 3306
```

```suggestion
TYPEORM_PORT = db_port
```
포트 넘버 또한 마찬가지입니다~! 3306은 mysql의 고유한 port number이기에 이를 특정할 수 있는 3306보다는 중의적인 단어로 대체해주세요~